src/globus_cli/commands/streams/__init__.py

```diff
@@ -0,0 +1,43 @@
+import typing as t
+
+from globus_cli.parsing import group
+
+
+def streams_flatten_response(in_doc: dict[str, t.Any]) -> dict[str, list[t.Any]]:
+    output_dict: dict[str, list[t.Any]] = {}
+    attributes_doc = in_doc.pop("attributes")
+    for i in attributes_doc:
+        output_dict[f"attributes_{i}"] = attributes_doc.get(i)
+    for i in in_doc:
+        output_dict[i] = in_doc.get(i)
```

Comment:
Is there a reason we wouldn't just mutate and return the original dict instead of constructing a new one?

Early on, we already mutate it by removing the "attributes" key so mutations are allowed.